app_utils/sections/experiment.py

```diff
@@ -1471,6 +1481,121 @@ async def experiment_download_model(q: Q, error: str = ""):
     await q.page.save()
 
 
+async def experiment_quantize_model(q: Q, error: str = ""):
+    experiment = q.client["experiment/display/experiment"]
+    experiment_path = q.client["experiment/display/experiment_path"]
+    logger.info(f"Quantizing model!")
+    zip_path = get_model_path(experiment.name, experiment_path)
+
+    if not os.path.exists(zip_path):
+        logger.info(f"Creating {zip_path} on demand")
+        cfg = load_config_yaml(os.path.join(experiment_path, "cfg.yaml"))
+
+        device = "cuda"
+
+        experiments = get_experiments(q)
+        num_running_queued = len(
+            experiments[experiments["status"].isin(["queued", "running"])]
+        )
+        if num_running_queued > 0:
+            logger.info("Waiting for queue to finish first.")
+            await q.page.save()
+            return
+
+        cfg, model, tokenizer = load_cfg_model_tokenizer(
+            experiment_path, merge=True, device=device
+        )
+
+        # Save model
```

Comment:
Not sure whether `save_pretrained` already detects when it was saved before, and skips fast?
Maybe wrap in a if-statement guard.